lib/factory_bot_rails/definition_file_paths.rb

```diff
@@ -19,5 +19,9 @@ def directories
     def files
       @files.select { |file| File.exist?(file) }
     end
+
+    def any?
+      directories.any?
```

Comment:
I think this will need to be `directories.any? || files.any?`. 

The case I am thinking of can be tested by adding `spec/fixtures/test/factories.rb` and

```rb
    context "a file exists but not a directory" do
      it "registers a reloader" do
        allow(reloader_class).to receive(:new)

        run_reloader(["spec/fixtures/test/factories", "not_exist_directory"])

        expect(reloader_class).to have_received(:new)
      end
    end
```

Reply:
Thanks! I fixed.